app/components/Receive/QRCodeForm/index.jsx

```diff
@@ -0,0 +1,109 @@
+// @flow
+import React from 'react'
+import classNames from 'classnames'
+
+import AssetInput from '../../Inputs/AssetInput'
+import NumberInput from '../../Inputs/NumberInput'
+import TextInput from '../../Inputs/TextInput'
+import Button from '../../Button/Button'
+import CopyToClipboard from '../../CopyToClipboard'
+import { Address } from '../../Blockchain'
+import { ASSETS, TOKENS } from '../../../core/constants'
+import { COIN_DECIMAL_LENGTH } from '../../../core/formatters'
+import GridIcon from '../../../assets/icons/grid.svg'
+
+import styles from './styles.scss'
+
+type Props = {
+  className?: string,
+  address: string,
+  onSubmit: Function
+}
+
+type State = {
+  asset: ?string,
+  amount: ?number,
+  description: ?string
+}
+
+export default class QRCodeForm extends React.Component<Props, State> {
+  image: ?HTMLImageElement
+
+  state = {
+    asset: undefined,
+    amount: undefined,
+    description: undefined
+  }
+
+  render() {
+    const { className, address, onSubmit } = this.props
+    const { asset, amount, description } = this.state
+    const symbols = ['-', ASSETS.NEO, ASSETS.GAS, ...Object.keys(TOKENS)]
```

Comment:
Same as comment above... seems to lead to an invalid option in the select input 
<img width="166" alt="screen shot 2018-09-18 at 7 12 29 pm" src="https://user-images.githubusercontent.com/13072035/45725193-e35f2a80-bb76-11e8-864e-d0dca529a3ed.png">